bsp/wavious-wh440/CMakeLists.txt

```diff
@@ -24,10 +24,16 @@ target_compile_options(
 )
 
 macro(bsp_post_build target)
```

Comment:
lets change target here to TARGET to match this change in wavious-mcu and what was done in the build script as well.  Will need to adjust through the rest of the macro as well.